shared/sbp.js

```diff
@@ -8,6 +8,12 @@ const globalFilters: Array<TypeFilter> = []
 const domainFilters: {[string]: Array<TypeFilter>} = {}
 const selectorFilters: {[string]: Array<TypeFilter>} = {}
 
+// Makes the `Buffer` global available in the browser.
+if (typeof window === 'object') {
+  const { Buffer } = require('buffer')
+  window.Buffer = Buffer
+}
+
```

Comment:
That's right, however I couldn't put it in `main.js` as no code is allowed before import statements, and it didn't really want to create a new file just for this.

Reply:
OK, so, here's what I recommend:

1. Create a file just for this called buffer.js or something with the word "buffer" in it.
2. Check to see if Buffer already exists. If so, do not do anything. This is so that if we ever switch back to rollup or something else, and that something else already includes Buffer, we won't import it twice.
3. Import it above sbp.js in main.js

Reply:
*Actually, why do you  have to import it before sbp.js?

Just import it afterward like all the imports. Import it near the top. sbp.js should be the first import always unless there's something that won't work otherwise.

Reply:
Would it be acceptable to put this snippet in `~/shared/functions.js`?

Reply:
Sure!